db/migrate/20230321081156_create_partners.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePartners < ActiveRecord::Migration[7.0]
+  def change
+    create_table :partners, id: :uuid do |t|
+      t.string :first_name
+      t.string :last_name
+      t.date :date_of_birth
+      t.boolean :has_national_insurance_number
+      t.string :national_insurance_number
+      t.references :legal_aid_application,index: true, type: :uuid
```

Comment:
Unless we have issues, we usually leave this to rails validation.  Because some of the models are created piecemeal by the forms adding chunks of data, and therefore break if we have null controls on the DB table, we have usually used that pattern on most of the tables.

Can you see an advantage/issue on this table that could be addressed?